pandas/tests/indexes/test_interval.py

```diff
@@ -491,6 +594,188 @@ def testcontains(self):
         assert not i.contains(20)
         assert not i.contains(-20)
 
+
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_interval_covers(self):
+
+        # class Interval:
+        #     def covers(self, other: Interval) -> bool
+        #     def covers(self, other: IntervalIndex) -> IntegerArray1D
+
+        assert     Interval(1, 3).covers(Interval(1.5, 2.5))
+        assert     Interval(1, 3).covers(Interval(1, 2))
+        assert     Interval(1, 3).covers(Interval(2, 3))
+        assert not Interval(1, 3).covers(Interval(0.5, 2.5))
+        assert not Interval(1, 3).covers(Interval(1.5, 3.5))
+
+        assert     Interval(1, 3, closed='right').covers(Interval(1, 3, closed='right'))
+        assert not Interval(1, 3, closed='right').covers(Interval(1, 3, closed='left'))
+        assert not Interval(1, 3, closed='right').covers(Interval(1, 3, closed='both'))
+
+        assert not Interval(1, 3, closed='left').covers(Interval(1, 3, closed='right'))
+        assert     Interval(1, 3, closed='left').covers(Interval(1, 3, closed='left'))
+        assert not Interval(1, 3, closed='left').covers(Interval(1, 3, closed='both'))
+
+        assert     Interval(1, 3, closed='both').covers(Interval(1, 3, closed='right'))
+        assert     Interval(1, 3, closed='both').covers(Interval(1, 3, closed='left'))
+        assert     Interval(1, 3, closed='both').covers(Interval(1, 3, closed='both'))
+
+        idx = IntervalIndex.from_tuples([(0, 1), (2, 3), (1, 3)])
```

Comment:
for clarity, add `closed='left'` or `closed='right'` here? (just so I don't need to remember the default